Parallels/ParallelsDesktop.download.recipe

```diff
@@ -9,6 +9,8 @@ For major versions 9 through 15, leave the PLATFORM_ARCH blank in your override.
 
 For major version 16, set the PLATFORM_ARCH in your override to either "intel" or "m1" depending on your desired architecture.
 
+For major version 17, set the PLATFORM_ARCH in your override to "image"
```

Comment:
Also need to add this note to ParallelsDesktop.pkg, ParallelsDesktop.munki, and ParallelsDesktop.install.